src/sandesh/coding_exam_1/ReturnIndicesTargetSum.java

```diff
@@ -0,0 +1,28 @@
+package sandesh.coding_exam_1;
+
+public class ReturnIndicesTargetSum {
+
+	int[] returnIndices(int[] numArr, int targetSum) {
+		int[] returnIndicesArray = new int[2];
+		for(int index=0; index < numArr.length; index++) {
+			for(int innerIndex=1; innerIndex<numArr.length; innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files

Reply:
Thank you @harshvegada for your valuable inputs. My code will not work in case it has to find target as 4 and the array is {2,2,3,4,5}.According to my logic it will return indices 0,0 as innerIndex starts from 1 always.